### PR TITLE
[AND-4375] Bugfix GDPR for nativeAd

### PR DESCRIPTION
**Pull request by HuFangshuai**
pr for bug ticket: https://vungle.atlassian.net/browse/AND-4375

### CODE REVIEW

File: Vungle/src/main/java/com/applovin/mediation/adapters/VungleMediationAdapter.java [hunk 1]
```diff
@@ -598,7 +599,7 @@ public void loadNativeAd(final MaxAdapterResponseParameters parameters, final Ac
         updateUserPrivacySettings( parameters );
 
         final Context applicationContext = getContext( activity );
-        nativeAd = new NativeAd( applicationContext, placementId );
+        nativeAd = new NativeAd( activity, placementId );
```
Comment (felixzhang0703) [inline Vungle/src/main/java/com/applovin/mediation/adapters/VungleMediationAdapter.java:601]: why do we use activity instead of applicationContext?

Comment (HuFangshuai) [reply Vungle/src/main/java/com/applovin/mediation/adapters/VungleMediationAdapter.java:601]: To show the GDPR dialog needs activity ref.

File: Vungle/src/main/java/com/applovin/mediation/adapters/VungleMediationAdapter.java [hunk 2]
```diff
@@ -1352,12 +1353,24 @@ public void prepareViewForInteraction(final MaxNativeAdView maxNativeAdView)
...
             nativeAdLayout.addView( mainView );
             maxNativeAdView.addView( nativeAdLayout );
 
+            View mediaView = getMediaView();
+            if (mediaView != null) {
+                if (mediaView.getParent() != null) {
+                    ((ViewGroup) mediaView.getParent()).removeView(mediaView);
+                }
+                ViewGroup contentViewGroup = maxNativeAdView.getMediaContentViewGroup();
+                contentViewGroup.removeAllViews();
```
Comment (felixzhang0703) [inline Vungle/src/main/java/com/applovin/mediation/adapters/VungleMediationAdapter.java:1379]: Is it possible to get a null view group?

**Review by felixzhang0703 — APPROVED** [review]
LGTM

File: Vungle/src/main/java/com/applovin/mediation/adapters/VungleMediationAdapter.java (hunk 2 above)
Comment (HuFangshuai) [reply Vungle/src/main/java/com/applovin/mediation/adapters/VungleMediationAdapter.java:1379]: yes, updated.

**Pull request merged**